torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -35,6 +35,9 @@ def __init__(
         self.fill = fill
 
     def _get_random_item(self, dct: Dict[K, V]) -> Tuple[K, V]:
+        # Question: This looks like a utility method that doesn't depend on self and could move out of this
+        # class to simplify its structure (for someone who tries to understand its API to implement a new method).
+        # Thoughts?
```

Comment:
I think this is a matter of style. In general, I tend to put utility methods that only are used by the class and its derivatives on the base class. There is no harm to move them to standalone functions. The only downside is that if someone outside of `torchvision` inherits from `_AutoAugmentBase` they don't have access to the functionality, since we don't expose it. No strong opinion here.

Reply:
I agree, your usage if valid. No strong opinion on my side either, the implementation can stay as-is.

@vfdev-5 any thoughts?

Reply:
Same for me, no strong option, the implementation can stay as-is

Reply:
Let's keep it then. Thanks both for the input.